content/news/release-20150424.md

```diff
@@ -0,0 +1,64 @@
+
+For modders and mappers, there have been quite a few important changes as well. We have introduced a new map format, to which maps running on this release will be updated automatically. All of the shared engine code has been moved into the OpenRA.Mods.Common project, so that third-party mods can drop their dependencies on our game mods. A lot of code has been changed to enable features that will be needed by the Tiberian Sun mod. One important change for Red Alert mappers is that the gem mine has been replaced with a new one. For more details, check out the [changelog](http://changelog.openra.net), as well as the [trait](https://github.com/openra/openra/wiki/Traits) and [Lua API](https://github.com/openra/openra/wiki/Lua-API) documentation.
+
+Looking forward, we hope to get the next release out faster than this time around (famous last words!). Feature-wise, you can probably expect the first appearance of the in-game map editor in the next release, in addition to much of the things that are currently in [our pull request queue](https://github.com/openra/openra/pulls). Beyond that, we will see... If you want to keep track of the development, keep an eye on the [release milestone](https://github.com/OpenRA/OpenRA/milestones/Next%20release) and the [development changelog](https://github.com/openra/openra/wiki/Changelog-(bleed).md).
+
+We will also try to make more use of [our YouTube channel](https://www.youtube.com/channel/UCRoiPL1J4K1-EhQeNazrYig) to show off new and little known existing features, provide mission walkthroughs and other content, so watch and subscribe! We will also feature the best user-submitted games.
+
+Now enough with the talking! Head on over to our [download section](/download/), grab the installer for your OS, and enjoy the game!
+
+<div style="text-align:center" markdown="1">
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/oK-paA7kRP0?rel=0" frameborder="0" allowfullscreen></iframe>
+
+The long awaited sandworms and Carryalls have finally arrived in our Dune 2000 mod!
+
+![Playable countries](/images/news/20150424-ra-country-dropdown.png)
+
+Red Alert now has playable country factions, and matching choices for random faction selection.
+
+![Overhauled server browser](/images/news/20150424-cnc-server-browser.png)
+
+The server browser now lists games from all available mods, and also allows joining games across mods.
+
+![Mission difficulty](/images/news/20150424-cnc-mission-difficulty.png)
+
+Some missions have received support for multiple difficulty levels.
+
+![Refund tooltip](/images/news/20150424-ra-refund-tooltip.png)
+
+In sell mode, the tooltip will now show a structure's expected refund value.
+
+![Tiberian Sun teaser](images/news/20150424-ts-teaser.png)
+
+Behind the scenes, the still unreleased, work-in-progress Tiberian Sun<br/>mod has received support for some crucial features.
```

Comment:
This wording feels a bit weird.  How about `Behind the scenes, we have made significant progress towards our work-in-progress Tiberian Sun mod.<br /> Keep an eye out for more news and media about TS on our Facebook page!`?